### ISSUE TITLE
[GlobalStep] Attempting to share Media from the device’s Gallery application causes a crash to occur.

### ISSUE THREAD

**Issue by wptester9845** [wordpress-mobile/WordPress-Android#11612]
### Description
Attempting to share Media from the device’s Gallery application to the WordPress application causes a crash to occur.

### Reproduction Rate
4/4 100%

### Expected behaviour
The Media should be shared to WordPress gracefully and without issue.

### Actual behaviour
The WordPress application crashes.

### Steps to reproduce the behaviour
1. Install WordPress 14.6
2. Login to a valid account.
3. Open the device’s "Gallery" application.
4. Attempt to share Media to the WordPress application.
##### Tested on the following
Samsung Galaxy S8+ (8.0)

#### Please see the attached video and crash log for more information
[AndroidGalleryShareCrash.zip](https://github.com/wordpress-mobile/WordPress-Android/files/4449729/AndroidGalleryShareCrash.zip)
[AndroidGalleryShareCrashLog.txt](https://github.com/wordpress-mobile/WordPress-Android/files/4449730/AndroidGalleryShareCrashLog.txt)

### Submitted by:
Luis Pimenta

**Comment by designsimply** [comment]
I tested and couldn't replicate using WPAndroid 14.6-rc-1 on Pixel 3 Android 10, but this is definitely worth a deeper look.

**Comment by planarvoid** [comment]
It looks like it's caused by this in the log: 
```
04-08 10:52:29.810 E/AndroidRuntime(26772): Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 2: TypedValue{t=0x2/d=0x7f0400e6 a=2}
04-08 10:52:29.810 E/AndroidRuntime(26772): 	at android.content.res.TypedArray.getColorStateList(TypedArray.java:538)
04-08 10:52:29.810 E/AndroidRuntime(26772): 	at androidx.cardview.widget.CardView.<init>(CardView.java:127)
04-08 10:52:29.810 E/AndroidRuntime(26772): 	at com.google.android.material.card.MaterialCardView.<init>(MaterialCardView.java:118)
04-08 10:52:29.810 E/AndroidRuntime(26772): 	at com.google.android.material.card.MaterialCardView.<init>(MaterialCardView.java:114)
```
Which (I think) is related to https://github.com/wordpress-mobile/WordPress-Android/issues/11616 
wdyt @khaykov ?

**Comment by khaykov** [comment]
Yes, it's the same issue, and the fix should be out already :)

**Comment by designsimply** [comment]
Fixed in #11623. Props @khaykov. 🎉
